### DISCUSSION TITLE
Consigli acquisto sensori

### DISCUSSION THREAD

**Discussion by H4Z4RD-H42** [indomus/forum#2216] [Hardware & Firmware]
Ciao raga, vengo direttamente al punto, vorrei realizzare un piccolo impianto di domotica per casa, composto da pochi elementi e quindi tutto gestito da HA.

Per il momento ho solo il termostato Tado°, una presa smart TP-Link, Google Nest e un Google Mini. Vorrei aggiungere un termometro ambientale in 3 camere, un sensore di apertura porta, un sensore di movimento in cucina, una striscia led in cucina e 3 prese smart di quelle che leggono i consumi.

Potete gentilmente indicarmi su quali prodotti devo orientarmi? Magari senza che questi necessitino di un hub per funzionare. Non cerco prodotti dalle mille funzioni, mi basta che facciano il loro, per esempio i termometri non devono per forza indicare pure umidità, previsioni e quant'altro, ma solo la temperatura.

Grazie!

**Comment by KC-inDomus** [comment]
personalmente ti consigliere [Lumi AQARA](https://indomus.it/componenti/lumi-linea-aqara-smart-home-automation/),una linea di sensori e attuatori zigbee molto economici ma al contempo molto affidabili e gradevoli esteticamente.

per implementarla su HA ti basta [questa antenna](https://indomus.it/componenti/itead-sonoff-zigbee-3-0-usb-dongle-plus-v2-zbdongle-e/) da usare installando [zigbee2mqtt](https://indomus.it/componenti/zigbee2mqtt-bridge-gateway-software-zigbee/).

**Reply by 50494554524F** [reply]
per bluetooth puoi guardare qui https://github.com/custom-components/ble_monitor quelli supportati al momento

(con le nuove versioni di home assistant vengono ormai riconosciuti automaticamente, se non lo sono ora lo saranno)

**Comment by H4Z4RD-H42** [comment]
Home Assistant nel mio caso è installato via Docker su un NAS QNAP, quindi non posso utilizzare l'antenna. Devo andare di Aqara Hub vero?

**Reply by KC-inDomus** [reply]
scusa, perché non potresti usare l'antenna?

**Reply by H4Z4RD-H42** [reply]
> scusa, perché non potresti usare l'antenna?

Ho riletto bene la guida, dimmi gentilmente se ho capito bene: l'antenna
è standalone e la procedura serve solo per flashare il FW e dirgli quindi
di comportarsi come un hub zigbee. Una volta terminato il flash, la posso
attaccare ovunque, per assurdo pure su una TV, basta che riesca ad
alimentarla. È corretto?

**Reply by snakuzzo** [reply]
No ...sbagliato.
L'antenna, o meglio...il coordinator, deve essere controllato da qualcosa. Quindi puoi collegarlo direttamente al NAS e farlo vedere al container docker che dovrà utilizzarlo.

Puoi usare zigbee2mqtt...e quindi la monterai per il container di zigbee2mqtt

Oppure puoi usare ZHA e quindi la monterai per il container di Core.

Guarda la guida di zigbee2mqtt per il QNAP abbiamo scritto come fare per farla vedere al container. Attenzione...guarda anche nei commenti perché pare che da QTS 5 sia necessario fare un altro giro per farla vedere al QNAP.

**Reply by snakuzzo** [reply]
Se poi vuoi posizionarlo dove ti pare dentro casa, dovresti usare un RPIZeroW ad esempio ed installare zigbee2mqtt lì sopra e collegarci il coordinator. In questo modo la sposti dove ti pare.

Oppure usare [ser2net](https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html), ma non è consigliato
